app/src/main/java/com/imdb_compose/MainActivity.kt

```diff
@@ -1,47 +1,1195 @@
+
+            canvas.drawRect(
+                left = leftPixel,
+                top = topPixel,
+                right = size.width + topPixel,
+                bottom = size.height + leftPixel,
+                paint = paint,
+            )
+        }
+    }
+)
+
+val gray100 = Color(0xff555555)
+val gray200 = Color(0xff444444)
+val gray300 = Color(0xff333333)
+val gray400 = Color(0xff222222)
+val gray500 = Color(0xff111111)
+val gray600 = Color(0xff000000)
+val blue100 = Color(0xff1eeeff)
+val blue200 = Color(0xff3D76E0)
+val blue300 = Color(0xff1e90ff)
+val blue400 = Color(0xff1e32ff)
+val bananaYellow = Color(0xffffe134)
+val ripeMango = Color(0xffffbf2e)
+val constuctionOrange = Color(0xffffc300)
+val deepSaffron= Color(0xffff9e28)
+val princetonOrange = Color(0xfffe7c22)
+val giantsOrange = Color(0xfffe5a1c)
+
+@Composable
+fun LazyRows(
+    viewModel: HomeScreenViewModel,
+    navController: NavController
+) {
+    viewModel.catagories.forEachIndexed { i, catagory ->
+        Box(
+            modifier = Modifier
+                .fillMaxWidth()
+                .shadow(
+                    color = gray600.copy(alpha = 0.6f),
+                    offsetX = 0.dp,
+                    offsetY = 0.dp,
+                    blurRadius = 4.dp
+                )
+        ) {
+            Row (
+                modifier = Modifier
+                    .fillMaxWidth(fraction = 0.95f)
+                    .fillMaxHeight(fraction = 0.95f)
+                    .shadow(
+                        color = gray400,
+                        offsetX = 8.dp,
+                        offsetY = 8.dp,
+                        blurRadius = 4.dp,
+                        blurRadiusFilter = "SOLID"
+                    )
+            ) {
+                Column (modifier = Modifier.padding(start = 8.dp, top = 8.dp)) {
+                    Row (
+                        modifier = Modifier.fillMaxWidth(),
+                        horizontalArrangement = Arrangement.SpaceBetween,
+                        verticalAlignment = Alignment.CenterVertically
+                    ) {
+                        // Orange Accent <Catagory>
+                        Row (verticalAlignment = Alignment.CenterVertically) {
+                            Box(
+                                modifier = Modifier
+                                    .padding(start = 8.dp)
+                                    .clip(RoundedCornerShape(24.dp))
+                                    .height(36.dp)
+                                    .width(6.dp)
+                                    .background(ripeMango)
+                            )
+                            Text(
+                                text = catagory,
+                                modifier = Modifier.padding(start = 8.dp),
+                                color = MaterialTheme.colorScheme.onSecondary,
+                                fontSize = MaterialTheme.typography.headlineMedium.fontSize,
```

Comment:
`textStyle` changes a lot of font attributes at once. Since you're using `headlineMedium` for all attributes, you can probably replace all these font attributes with one call to `textStyl = MaterialTheme.typography.headlineMedium`

You should do this everywhere else you make multiple calls to change font attributes